lib/msal-angular/docs/v2-docs/errors.md

```diff
@@ -25,7 +25,7 @@ In `@azure/msal-angular` there are 2 common scenarios when this can happen:
 
 We recommend that you bootstrap the the **`MsalRedirectComponent`** if you want to use redirects as this will handle all redirects for you. See the docs on redirects [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular/docs/v2-docs/redirects.md) for how to use the `MsalRedirectComponent` or alternatively calling `handleRedirectObservable` manually. 
 
-We also recommend that any interaction should be done after subscribing to the `inProgress$` subject and filtering for none. Please see our [events doc](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/events.md#the-inprogress-observable) for more details. 
+We also recommend that any interaction should be done after subscribing to the `inProgress$` observable and filtering for `InteractionStatus.None`. Please see our [events doc](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/events.md#the-inprogress-observable) for more details. 
```

Comment:
Let's be more explicit that attempting interaction while another is in progress is not supported and checking for `InteractionStatus.None` is how you ensure you don't do this on accident.

Reply:
Makes sense. Updated.